tests/x11/desktop_mainmenu.pm

```diff
@@ -18,6 +18,7 @@ use testapi;
 
 
 sub run {
+    sleep 5;
```

Comment:
Can you please add a comment what this sleep is for?

Reply:
Added a comment in follow-up #4177